.changelog/2163.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+`manifest/provider/apply.go`: update flow of wait timeout
```

Comment:
This doesn't say much about the bug that has been fixed and the affected resource.

I would propose something like this:

```release-note:bug
`resource/kubernetes_manifest`: update the `wait` block flow to address an issue where a Terraform apply operation fails because it does not meet the required conditions, causing the target resource to exist on the Kubernetes end but not in the Terraform state file. As a result, subsequent apply operations fail due to the resource already existing. The proposed fix ensures that the state file is updated when the resource is created, resolving this problem.
```

Doesn't have to be exactly this, but should give practitioners an understanding of the change, like which resource is affected and how.